tensorflow/python/ops/histogram_ops.py

```diff
@@ -69,6 +70,9 @@ def histogram_fixed_width_bins(values,
   >>> indices.numpy()
   array([0, 0, 1, 2, 4, 4], dtype=int32)
   """
+  if nbins < 0:
```

Comment:
This sometimes errors with

```
  File ".../tensorflow/python/histogram_ops_test_cpu.runfiles/org_tensorflow/tensorflow/python/ops/histogram_ops.py", line 144, in histogram_fixed_width
    if nbins < 0:
TypeError: '<' not supported between instances of 'list' and 'int'
```

Reply:
This does not go to the C++ implementation, so it needs to be handled here,but needs to validate that the argument is either a number (int) or a scalar tensor of integral dtype.

Reply:
This was throwing error due to the condition at line 144, now I have removed it since it is being handled in C++. 
nbins:  Scalar `int32 Tensor`.  Number of histogram bins.

Reply:
This doesn't call the function that includes line 144, so it was not giving an error due to it.

Reply:
error says `line 144, in histogram_fixed_width File ".../tensorflow/python/histogram_ops_test_cpu.runfiles/org_tensorflow/tensorflow/python/ops/histogram_ops.py",`